tools/test_arm.cpp

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
I don't really get why this test is here, separated from the regular tests. If this is a test that's handy to have around but can't run as part of the test suite, I would prefer it still lives under tests but with an explicit build target (like test_upnp and test_natpmp)